src/app/pages/categories/management/pages-categories-management.component.html

```diff
@@ -0,0 +1,82 @@
+    <mat-spinner [diameter]="30"></mat-spinner>
+  </div>
+
+  <div class="message-container" *ngIf="transactionCategoriesData.hasError">
+    <app-system-message [type]="notificationTypes.Error"
+                        text="Something went wrong. We were not able to get your wallets"
+                        description="Please try refreshing your browser"></app-system-message>
+  </div>
+
+  <div class="categories__container" *ngIf="transactionCategoriesData.data">
+
+    <h2 class="categories__title">
+        Categories Management
+    </h2>
+
+    <div class="categories__sort__add">
+      <div class="categories__sort">
+        <mat-form-field>
+          <mat-label>Select type</mat-label>
+          <mat-select [formControl]="categoriesTypeForm">
+            <mat-option *ngFor="let type of selectTransactionCategoriesTypes | keyvalue" [value]="type.value">
+              {{ type.key }}
+            </mat-option>
+          </mat-select>
+        </mat-form-field>
+      </div>
+
+      <div class="category__add">
+        <button mat-flat-button color="primary" class="income">Add income category</button>
+      </div>
+
+      <div class="category__add">
+        <button mat-flat-button color="primary" class="expense">Add expense category</button>
+      </div>
+
+    </div>
+
+    <div class="no-results-message-container" *ngIf=" displayedCategories.length === 0">
+      <app-system-message [type]="notificationTypes.Error"
+                          text="NO RESULTS"
+                          description="Please add first category"></app-system-message>
+    </div>
+
+    <div *ngIf="displayedCategories.length > 0" class="table__container">
+      <table mat-table [dataSource]="displayedCategories" class="mat-elevation-z8">
+
+        <ng-container matColumnDef="name">
+          <th mat-header-cell *matHeaderCellDef> Name </th>
+          <td mat-cell *matCellDef="let element"> {{ element.name }} </td>
+        </ng-container>
+
+        <ng-container matColumnDef="type">
+          <th mat-header-cell *matHeaderCellDef> Type </th>
+          <td mat-cell *matCellDef="let element">
+            <span [ngClass]="element.type === categoryType.Expense ? 'type--expense' : 'type--income'">
+              {{ element.type }}
+            </span>
+          </td>
+        </ng-container>
+
+        <ng-container matColumnDef="actions">
+          <th mat-header-cell *matHeaderCellDef> Actions </th>
+          <td mat-cell *matCellDef="let element">
+            <button mat-stroked-button>
+              <mat-icon aria-hidden="false" fontIcon="edit"></mat-icon>
+            </button>
+            <button mat-raised-button color="warn">
+              <mat-icon aria-hidden="false" fontIcon="delete"></mat-icon>
+            </button>
+          </td>
+        </ng-container>
+
+        <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+        <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
+      </table>
```

Comment:
Today this was merged https://github.com/bycza-zagroda/track-expenses-app-frontend/pull/103.
Here we should also use `<app-no-results>` component in case of no categories existing.